R/utils.R

```diff
@@ -65,3 +65,7 @@ get_list_element <- function(x, names) {
   for (i in names[seq_len(n - 1)]) if (!is.list(x <- x[[i]])) return()
   x[[names[n]]]
 }
+
+pkg_file <- function(...) {
+  system.file(..., package = "rticles")
+}
```

Comment:
That's great! In the future, I think you'll have several more chances to factor out code in our other packages.

BTW, for simple functions like this, I'd write on one line, too:

```suggestion
pkg_file <- function(...) system.file(..., package = "rticles")
```

I have this habit of "saving vertical space as much as possible" because when I read code, I don't want to scroll up and down frequently. If that doesn't make sense to you, let's follow whatever coding style that you are comfortable with.

Reply:
I am used to have all the inside content of a function aligned at one indent, so I do this even for a one liner. 
But I'll see if I get used to this because I see the advantage of having less lines vertically and for this short function it makes sense to use one line.